lablqml/src_tests/test3.qml

```diff
@@ -0,0 +1,45 @@
+import QtQuick 2.1
+import QtQuick.Controls 1.0
+
+/* Demonstraction of direct property binding
+OCaml binds to objects named "test" and "mirror".
+When test changes, the binding function is called and the
+mirror's text value is set */
+
+ApplicationWindow {
+    visible: true
+    QtObject {
+	id: t
+	objectName: "test"
+	property string msg: "Hello world! Hello Qml with OCaml!"
+
+	// QtObjects can only be nested as properties
```

Comment:
I'm not sure, I'd need to investigate. QtObject is meant to be a simple, plain, data object; it doesn't have `children : list<Item>` etc. So it behaves very differently. I've only used it to create structures of properties.